cmd/object-api-utils.go

```diff
@@ -160,6 +157,28 @@ func IsValidObjectPrefix(object string) bool {
 	return true
 }
 
+// objectNameLength -check for the length of object name
+func objectNameLength(bucket, object string) error {
```

Comment:
Can the function name be something like `validObjectNameLength`? just `objectNameLength` makes it sound like this function will return `length of the object name`

Reply:
changed